pkg/networkmanager/index.html

```diff
@@ -104,27 +104,30 @@ <h1 id="testing-connection-curtain-restoring" translate="yes">Restoring connecti
         {{! member interfaces will be rendered into here as a list of checkboxes }}
       </div>
 
-      <label class="control-label" for="network-bridge-settings-stp-enabled-input" translate="yes">Spanning tree protocol (STP)</label>
+      <label class="control-label" for="network-bridge-settings-stp-enabled-input" translate="yes">Topology</label>
```

Comment:
I think STP is a protocol - not a Network topology. But please ask someone else on that.

Reply:
Docs I've read about it say it's related to network topology. :shrug: Protocol is also fine.

Reply:
Marius was on the opinion that all these terms are too raw and we should rather leave it Options. I also proposed 'Additional options'. Let's pick up one of the two, I trust marius on this.

Reply:
Here are some examples:

https://www.tutorialspoint.com/spanning-tree-protocol

> Spanning Tree Protocol (STP) is a communication protocol operating at data link layer the OSI model to prevent bridge loops and the resulting broadcast storms. It creates a loop − free topology for Ethernet networks.

https://en.wikipedia.org/wiki/Spanning_Tree_Protocol

> The Spanning Tree Protocol (STP) is a network protocol that builds a loop-free logical topology for Ethernet networks.

https://techacad.net/introduction-to-spanning-tree-protocol/

> Spanning Tree Protocol was invented to create a loop free topology by blocking interfaces.

---

But we could call it protocol or whatever too. I was trying to use a word that more specifically explains the intent of STP, rather than going with something less specific like "protocol" (which is also redundant, as that's what the P stands for) or vague like "options", which we already decided against at the beginning of this PR.

However, the goal is to move the label to actually be a label for the checkbox. The side label's value is less important, so as long as you're happy with the term, I'm happy.

Reply:
What about:

```
Spanning tree protocol:  [ ] Enable
```

Reply:
We could also consider "Fault tolerance" :shrug:

Reply:
Which turns into this when enabled:

```
Spanning tree protocol:  [x] Enable
                         Priority: ____
                         Forward delay: ___
                         ...
```

Reply:
@mvollmer: No.

Firstly, that defeats the entire goal of this PR.

Secondly, that's like saying "enable/disable enable", which makes no sense.

See this recent twitter thread:
https://twitter.com/hbons/status/1434810758974648324

Screenshot if you don't want to visit twitter:

![image](https://user-images.githubusercontent.com/10246/133226825-d42cf9cd-6a93-4907-940a-bda457592afb.png)

Reply:
Then leave "Options".